diff --git a/BUILD_DIR/Makefile.in b/BUILD_DIR/Makefile.in
index 72fce22..c4c16f1 100644
--- a/BUILD_DIR/Makefile.in
+++ b/BUILD_DIR/Makefile.in
@@ -11,7 +11,7 @@ all local: check-selection hd_install
 	@echo "Finished make $@" >&2
 
 hd_install: hd_install.c
-	${CC} -o $@ hd_install.c
+	@echo "SKIP"
 
 install: check-selection
 	@${MAKE} ${HD_SELECTED} HD_TARGET=$@
diff --git a/BUILD_DIR/configure.in b/BUILD_DIR/configure.in
index 7ba0458..80708ea 100644
--- a/BUILD_DIR/configure.in
+++ b/BUILD_DIR/configure.in
@@ -888,260 +888,7 @@ fi
 
 AC_MSG_NOTICE([Preliminary check of compiler choices...])
 
-APPLEXCODE="no"
-# Existence of CC:
-if test "x$CC" != x; then
-  AC_MSG_NOTICE([Checking your C compiler setting ($CC)])
-  if test ! -x $CC; then
-    # Allow for $CC to have been set without a path:
-    AC_CHECK_PROG(CCTEST,$CC,yes,no)
-    if test "x$CCTEST" = xno; then
-      AC_MSG_ERROR($CC is unusable or does not exist)
-    fi
-  fi
-  # Check support for XCode clang version:
-  if test `echo $host | grep -c apple-darwin` -ne 0; then
-    if test `$CC --version | grep -ci 'Apple'` -ne 0; then
-      APPLEXCODE="yes"
-      # PLACEHOLDER now since clang_supported=yes for all components:
-      if test "x$clang_supported" = xno; then
-        AC_MSG_CHECKING([whether Apple XCode $CC is supported for your selected components])
-        changequote(,)
-        xcode_new=`$CC -dumpversion | grep -c '^[1][23456789]\.[0123456789]'`
-        changequote([,])
-        if test $xcode_new -eq 0; then
-          AC_MSG_RESULT(yes)
-        else
-          AC_MSG_RESULT(no)
-          AC_MSG_WARN(This compiler is not yet supported for ____)
-          AC_MSG_WARN(Visit the following link for more information:)
-          AC_MSG_WARN(http://heasarc.gsfc.nasa.gov/lheasoft/macos.html)
-          AC_MSG_ERROR(Please choose a different C compiler)
-        fi
-      fi
-    fi
-    # Check compiler architecture:
-    AC_MSG_CHECKING([architecture of $CC])
-    echo "int main(){return(0);}" > /tmp/$$.c
-    $CC -v -o /tmp/$$.o /tmp/$$.c 2> /tmp/$$.log
-    if test -f /tmp/$$.o; then
-      if test `file /tmp/$$.o | grep -ci 'x86_64'` -ne 0; then
-        AC_MSG_RESULT(x86_64)
-        # Disallow Intel compiler for M1 build:
-        if test `echo $host | grep -c aarch64` -ne 0; then
-          AC_MSG_NOTICE(arm64 build requires arm64 compilers)
-          AC_MSG_ERROR(Please choose an arm64 C compiler or use the 'mac_intel_build' option instead)
-        fi
-      elif test `file /tmp/$$.o | grep -ci 'arm64'` -ne 0; then
-        AC_MSG_RESULT(arm64)
-        # Disallow M1 compiler for Intel build, unless Rosetta can be used:
-        if test `echo $host | grep -c x86_64` -ne 0; then
-          if test "x$APPLEXCODE" = xno; then
-            AC_MSG_WARN(An Intel/x86_64 build requires x86_64 compilers)
-            AC_MSG_ERROR(Please choose an x86_64 C compiler)
-          else
-            # Try Rosetta flag for clang:
-            $CC -v -arch x86_64 -o /tmp/$$b.o /tmp/$$.c 2> /tmp/$$b.log
-            if test -f /tmp/$$b.o; then
-              if test `file /tmp/$$b.o | grep -ci 'x86_64'` -ne 0; then
-                AC_MSG_NOTICE(Will use '-arch x86_64' Rosetta flag for $CC)
-              fi
-            else
-              AC_MSG_ERROR(Simple Rosetta test of $CC failed)
-            fi
-          fi
-        fi
-      else
-        AC_MSG_RESULT(unknown...this may not work...)
-      fi
-    else
-      AC_MSG_ERROR(Simple test of $CC failed)
-    fi
-    rm -rf /tmp/$$*
-  fi
-  # Check functionality:
-  AC_PROG_CC
-else
-  AC_MSG_NOTICE([CC not set, will look for a C compiler downstream.])
-fi
-
-APPLEXCODE="no"
-# Existence of CXX:
-if test "x$CXX" != x; then
-  AC_MSG_NOTICE([Checking your C++ compiler setting ($CXX)])
-  if test ! -x $CXX; then
-    # Allow for $CXX to have been set without a path:
-    AC_CHECK_PROG(CXXTEST,$CXX,yes,no)
-    if test "x$CXXTEST" = xno; then
-      AC_MSG_ERROR($CXX is unusable or does not exist)
-    fi
-  fi
-  # APPLE
-  if test `echo $host | grep -c apple-darwin` -ne 0; then
-    if test `$CXX --version | grep -ci 'Apple'` -ne 0; then
-      APPLEXCODE="yes"
-    fi
-    # Check compiler architecture:
-    AC_MSG_CHECKING([architecture of $CXX])
-    echo "int main(){return(0);}" > /tmp/$$.cxx
-    $CXX -v -o /tmp/$$.o /tmp/$$.cxx 2> /tmp/$$.log
-    if test -f /tmp/$$.o; then
-      if test `file /tmp/$$.o | grep -ci 'x86_64'` -ne 0; then
-        AC_MSG_RESULT(x86_64)
-        # Disallow Intel compiler for M1 build:
-        if test `echo $host | grep -c aarch64` -ne 0; then
-          AC_MSG_NOTICE(arm64 build requires arm64 compilers)
-          AC_MSG_ERROR(Please choose an arm64 C++ compiler or use the 'mac_intel_build' option instead)
-        fi
-      elif test `file /tmp/$$.o | grep -ci 'arm64'` -ne 0; then
-        AC_MSG_RESULT(arm64)
-        # Disallow M1 compiler for Intel build, unless Rosetta can be used:
-        if test `echo $host | grep -c x86_64` -ne 0; then
-          if test "x$APPLEXCODE" = xno; then
-            AC_MSG_WARN(An Intel/x86_64 build requires x86_64 compilers)
-            AC_MSG_ERROR(Please choose an x86_64 C++ compiler)
-          else
-            # Try Rosetta flag for clang++:
-            $CXX -v -arch x86_64 -o /tmp/$$b.o /tmp/$$.cxx 2> /tmp/$$b.log
-            if test -f /tmp/$$b.o; then
-              if test `file /tmp/$$b.o | grep -ci 'x86_64'` -ne 0; then
-                AC_MSG_NOTICE(Will use '-arch x86_64' Rosetta flag for $CXX)
-              fi
-            else
-              AC_MSG_ERROR(Simple Rosetta test of $CC failed)
-            fi
-          fi
-        fi
-      else
-        AC_MSG_RESULT(unknown...this may not work...)
-      fi
-    else
-      AC_MSG_RESULT(Simple test of $CXX failed)
-    fi
-    rm -rf /tmp/$$*
-  fi
-  # Check functionality:
-  AC_PROG_CXX
-else
-  AC_MSG_NOTICE([CXX not set, will look for a C++ compiler downstream.])
-fi
-
-# Existence of FC:
-if test "x$FC" != x; then
-  AC_MSG_NOTICE([Checking your Fortran compiler setting ($FC)])
-  if test ! -x $FC; then
-    # Allow for $FC to have been set without a path:
-    AC_CHECK_PROG(FCTEST,$FC,yes,no)
-    if test "x$FCTEST" = xno; then
-      AC_MSG_ERROR($FC is unusable or does not exist)
-    fi
-  fi
-  # APPLE
-  if test `echo $host | grep -c apple-darwin` -ne 0; then
-    # Check compiler architecture:
-    AC_MSG_CHECKING([architecture of $FC])
-    echo "      program fortrantest" > /tmp/$$.f
-    echo "      end" >> /tmp/$$.f
-    $FC -v -o /tmp/$$.o /tmp/$$.f 2> /tmp/$$.log
-    if test -f /tmp/$$.o; then
-      if test `file /tmp/$$.o | grep -ci 'x86_64'` -ne 0; then
-        AC_MSG_RESULT(x86_64)
-        # Disallow Intel compiler for M1 build:
-        if test `echo $host | grep -c aarch64` -ne 0; then
-          AC_MSG_NOTICE(arm64 build requires arm64 compilers)
-          AC_MSG_ERROR(Please choose an arm64 Fortran compiler or use the 'mac_intel_build' option instead)
-        fi
-      elif test `file /tmp/$$.o | grep -ci 'arm64'` -ne 0; then
-        AC_MSG_RESULT(arm64)
-        # Disallow M1 compiler for Intel build:
-        if test `echo $host | grep -c x86_64` -ne 0; then
-          AC_MSG_WARN(An Intel/x86_64 build requires x86_64 compilers)
-          AC_MSG_ERROR(Please choose an x86_64 Fortran compiler)
-        fi
-      else
-        AC_MSG_RESULT(unknown...this may not work...)
-      fi
-    else
-      AC_MSG_ERROR(Simple test of $FC failed)
-    fi
-    rm -rf /tmp/$$*
-  fi
-  # Check functionality:
-  AC_PROG_FC
-else
-  AC_MSG_NOTICE([FC not set, will look for a Fortran compiler downstream.])
-fi
-
-# PERL:
-if test "x$PERL" != x; then
-  AC_MSG_NOTICE([Checking your Perl setting ($PERL)])
-  if test ! -x $PERL; then
-    # Allow for $PERL to have been set without a path:
-    AC_CHECK_PROG(PERLTEST,$PERL,yes,no)
-    if test "x$PERLTEST" = xno; then
-      AC_MSG_ERROR($PERL is unusable or does not exist)
-    fi
-  fi
-  # if we're on an arm64 MAC
-  if test `echo $actual_host | grep -c aarch64-apple` -ne 0; then
-    # if build arch is x86_64:
-    if test `echo $host | grep -c x86_64-apple` -ne 0; then
-      # if the perl identifies itself as arm64, error
-      if test `file \`which $PERL\` | grep -ci 'arm64'` -ne 0; then
-        AC_MSG_WARN(An Intel/x86_64 build requires x86_64 Perl)
-        AC_MSG_ERROR(Please choose an x86_64 Perl)
-      fi
-    # if build arch is arm64:
-    elif test `echo $host | grep -c aarch64-apple` -ne 0; then
-      # if the perl does NOT identify itself as arm64, error
-      if test `file \`which $PERL\` | grep -ci 'arm64'` -eq 0; then
-        AC_MSG_NOTICE(arm64 build requires an arm64 Perl)
-        AC_MSG_ERROR(Please choose an arm64-compatible Perl)
-      fi
-      AC_MSG_NOTICE(ok)
-    fi
-  else
-    AC_MSG_NOTICE(ok)
-  fi
-fi
-
-# PYTHON:
-if test "x$PYTHON" != x; then
-  AC_MSG_NOTICE([Checking your Python setting ($PYTHON)])
-  if test ! -x $PYTHON; then
-    # Allow for $PYTHON to have been set without a path:
-    AC_CHECK_PROG(PYTHONTEST,$PYTHON,yes,no)
-    if test "x$PYTHONTEST" = xno; then
-      AC_MSG_ERROR($PYTHON is unusable or does not exist)
-    fi
-  fi
-  # if we're on an arm64 MAC
-  if test `echo $actual_host | grep -c aarch64-apple` -ne 0; then
-    # allow for python as a script
-    if test `file \`which $PYTHON\` | grep -ci 'ascii'` -ne 0; then
-      AC_MSG_RESULT($PYTHON is non-binary)
-    else
-      # if build arch is x86_64:
-      if test `echo $host | grep -c x86_64-apple` -ne 0; then
-        # if the python identifies itself as arm64, error
-        if test `file \`which $PYTHON\` | grep -ci 'arm64'` -ne 0; then
-          AC_MSG_WARN(An Intel/x86_64 build requires x86_64 Python)
-          AC_MSG_ERROR(Please choose an x86_64 Python)
-        fi
-      # if build arch is arm64:
-      elif test `echo $host | grep -c aarch64-apple` -ne 0; then
-        # if the python does NOT identify itself as arm64, error
-        if test `file \`which $PYTHON\` | grep -ci 'arm64'` -eq 0; then
-          AC_MSG_NOTICE(arm64 build requires an arm64 Python)
-          AC_MSG_ERROR(Please choose an arm64-compatible Python)
-        fi
-      fi
-      AC_MSG_NOTICE(ok)
-    fi
-  else
-    AC_MSG_NOTICE(ok)
-  fi
-fi
+APPLEXCODE="yes"
 
 #-------------------------------------------------------------------------------
 # Recursively configure all selected components.
@@ -1151,6 +898,7 @@ for h_dir in $HD_SELECTED_DIRS; do
   h_config_heacore=`$hd_scanenv $h_dir/$hd_conf_file HEACORE`
   h_config_heatools=`$hd_scanenv $h_dir/$hd_conf_file HEATOOLS`
   h_config_cmd=`$hd_scanenv $h_dir/$hd_conf_file CONFIGURE_CMD`
+  h_config_cmd="$h_config_cmd --host=${target}"
   if test $? -ne 0 -o "x$h_config_cmd" = x; then
     AC_MSG_ERROR(Unable to determine how to run configure for $h_comp)
   fi
diff --git a/Xspec/BUILD_DIR/Makefile.in b/Xspec/BUILD_DIR/Makefile.in
index 225bd33..164e1f4 100644
--- a/Xspec/BUILD_DIR/Makefile.in
+++ b/Xspec/BUILD_DIR/Makefile.in
@@ -14,8 +14,6 @@ HD_CLEAN		= ${HD_BLD_EXEC_PFX} ${HD_BLD_PFX}
 
 HD_BLD_CTASK		= hd_install
 
-HD_BLD_CTASK_SRC_c	= hd_install.c
-
 #HD_CFLAGS               = ${CFLAGS}
 
 HD_INSTALL_TASKS	= hd_install
diff --git a/Xspec/BUILD_DIR/configure.in b/Xspec/BUILD_DIR/configure.in
index 8183d8e..7d49f61 100644
--- a/Xspec/BUILD_DIR/configure.in
+++ b/Xspec/BUILD_DIR/configure.in
@@ -632,15 +632,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -670,15 +661,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -728,15 +710,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
@@ -749,23 +722,6 @@ AC_SUBST(GXX)
 if test "x$CXX" != "x:"; then
   AC_LANG_PUSH([C++])
 
-  # Test to see whether the compiler has a std::count defect (Solaris)
-  AC_CACHE_CHECK(whether the compiler has a std::count defect,ac_cv_cxx_std_count_defect,
-   AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[#include <functional>
-	#include <vector>
-	#include <algorithm>
-        using namespace std;]],
-        [[vector<int> test(10,1);
-        int i1 (count(test.begin(),test.end(),1));
-        int gr1 (count_if(test.begin(),test.end(),bind2nd(greater<int>(),0)));
-        return 0;
-   ]])],
-   [ac_cv_cxx_std_count_defect=no],[ac_cv_cxx_std_count_defect=yes])
-  )
-  if test "$ac_cv_cxx_std_count_defect" = yes; then
-    AC_DEFINE(STD_COUNT_DEFECT,1,[define if the compiler has a std::count defect])
-  fi
-
   LDFLAGS_SAVE="${LDFLAGS}"
   LDFLAGS="${LDFLAGS} -std=c++11"
   AC_CACHE_CHECK(whether the host requires -lpthread,ac_cv_cxx_pthread,
diff --git a/attitude/BUILD_DIR/configure.in b/attitude/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/attitude/BUILD_DIR/configure.in
+++ b/attitude/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/demo/BUILD_DIR/configure.in b/demo/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/demo/BUILD_DIR/configure.in
+++ b/demo/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/ftools/BUILD_DIR/Makefile.in b/ftools/BUILD_DIR/Makefile.in
index 29cd139..d299be6 100644
--- a/ftools/BUILD_DIR/Makefile.in
+++ b/ftools/BUILD_DIR/Makefile.in
@@ -14,8 +14,6 @@ HD_CLEAN		= ${HD_BLD_EXEC_PFX} ${HD_BLD_PFX}
 
 HD_BLD_CTASK		= hd_install
 
-HD_BLD_CTASK_SRC_c	= hd_install.c
-
 #HD_CFLAGS               = ${CFLAGS}
 
 HD_INSTALL_TASKS	= hd_install
diff --git a/ftools/BUILD_DIR/configure.in b/ftools/BUILD_DIR/configure.in
index a984e9f..bb156c1 100644
--- a/ftools/BUILD_DIR/configure.in
+++ b/ftools/BUILD_DIR/configure.in
@@ -700,15 +700,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -758,15 +749,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -826,15 +808,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/heacore/BUILD_DIR/Makefile.in b/heacore/BUILD_DIR/Makefile.in
index d24d179..0c3ddc2 100644
--- a/heacore/BUILD_DIR/Makefile.in
+++ b/heacore/BUILD_DIR/Makefile.in
@@ -14,8 +14,6 @@ HD_CLEAN		= ${HD_BLD_EXEC_PFX} ${HD_BLD_PFX}
 
 HD_BLD_CTASK		= hd_install
 
-HD_BLD_CTASK_SRC_c	= hd_install.c
-
 #HD_CFLAGS               = ${CFLAGS}
 
 HD_INSTALL_TASKS	= hd_install
diff --git a/heacore/BUILD_DIR/configure.in b/heacore/BUILD_DIR/configure.in
index 958c18c..341a20d 100644
--- a/heacore/BUILD_DIR/configure.in
+++ b/heacore/BUILD_DIR/configure.in
@@ -431,15 +431,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -491,15 +482,6 @@ if test "x$FC" = x; then
 else
   F77="$FC"
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -565,15 +547,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
@@ -2025,6 +1998,7 @@ for h_component in $h_components; do
       if test $? -ne 0 -o "x$h_config_cmd" = x; then
         AC_MSG_ERROR(Unable to determine how to run configure for $h_component)
       fi
+      h_config_cmd="$h_config_cmd --host=${target}"
       AC_MSG_RESULT(--------------------------------------------------------------------------------)
       eval echo \"$h_config_cmd\"
       (eval $h_config_cmd)
diff --git a/heagen/BUILD_DIR/configure.in b/heagen/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/heagen/BUILD_DIR/configure.in
+++ b/heagen/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/heasim/BUILD_DIR/configure.in b/heasim/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/heasim/BUILD_DIR/configure.in
+++ b/heasim/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/heasptools/BUILD_DIR/configure.in b/heasptools/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/heasptools/BUILD_DIR/configure.in
+++ b/heasptools/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/heatools/BUILD_DIR/configure.in b/heatools/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/heatools/BUILD_DIR/configure.in
+++ b/heatools/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/hitomixrism/BUILD_DIR/configure.in b/hitomixrism/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/hitomixrism/BUILD_DIR/configure.in
+++ b/hitomixrism/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/integral/BUILD_DIR/configure.in b/integral/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/integral/BUILD_DIR/configure.in
+++ b/integral/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/ixpe/BUILD_DIR/configure.in b/ixpe/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/ixpe/BUILD_DIR/configure.in
+++ b/ixpe/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/maxi/BUILD_DIR/configure.in b/maxi/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/maxi/BUILD_DIR/configure.in
+++ b/maxi/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/nicer/BUILD_DIR/configure.in b/nicer/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/nicer/BUILD_DIR/configure.in
+++ b/nicer/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/nustar/BUILD_DIR/configure.in b/nustar/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/nustar/BUILD_DIR/configure.in
+++ b/nustar/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/suzaku/BUILD_DIR/configure.in b/suzaku/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/suzaku/BUILD_DIR/configure.in
+++ b/suzaku/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/swift/BUILD_DIR/configure.in b/swift/BUILD_DIR/configure.in
index 81e3a04..99b2533 100644
--- a/swift/BUILD_DIR/configure.in
+++ b/swift/BUILD_DIR/configure.in
@@ -710,15 +710,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -748,15 +739,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -806,15 +788,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
diff --git a/tcltk/BUILD_DIR/Makefile.in b/tcltk/BUILD_DIR/Makefile.in
index 8e0e681..603f475 100644
--- a/tcltk/BUILD_DIR/Makefile.in
+++ b/tcltk/BUILD_DIR/Makefile.in
@@ -13,8 +13,6 @@ HD_CLEAN		= ${HD_BLD_EXEC_PFX} ${HD_BLD_PFX} \
 
 HD_BLD_CTASK		= hd_install
 
-HD_BLD_CTASK_SRC_c	= hd_install.c
-
 #HD_CFLAGS               = ${CFLAGS}
 
 HD_INSTALL_TASKS	= hd_install
diff --git a/tcltk/BUILD_DIR/configure.in b/tcltk/BUILD_DIR/configure.in
index 2805950..5cbba25 100644
--- a/tcltk/BUILD_DIR/configure.in
+++ b/tcltk/BUILD_DIR/configure.in
@@ -519,15 +519,6 @@ AC_PROG_CC([$c_search_order])
 if test "x$CC" = x; then
   AC_MSG_ERROR(Could not find a C compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple C executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GCC" = x; then
   GCC=no
 fi
@@ -557,15 +548,6 @@ AC_PROG_FC([$f_search_order ifort g95 f95 f90])
 if test "x$FC" = x; then
   AC_MSG_ERROR(Could not find a Fortran compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple Fortran executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 
 FCFLAGS=$FFLAGS
 FC_BASENAME=`basename $FC`
@@ -615,15 +597,6 @@ if test "x$CXX" = x; then
 elif test "x$CXX" = "xg++" -a "x$GXX" != xyes; then
   AC_MSG_ERROR(Could not find a C++ compiler in $PATH)
 fi
-if test "$cross_compiling" = yes; then
-  AC_MSG_WARN(Cannot run a simple G++ executable on your system:)
-  AC_MSG_WARN(There may be something wrong with your compiler,)
-  AC_MSG_WARN(or perhaps you are trying to cross-compile?)
-  AC_MSG_WARN(Cross-compiling is not supported within HEADAS.)
-  AC_MSG_WARN(Please make sure your compiler is working.)
-  AC_MSG_WARN(Contact the FTOOLS help desk for further assistance.)
-  AC_MSG_ERROR(Cross-compiling is not allowed.)
-fi
 if test "x$GXX" = x; then
   GXX=no
 fi
