diff --git a/heacore/BUILD_DIR/configure.in b/heacore/BUILD_DIR/configure.in
index 958c18c..2364087 100644
--- a/heacore/BUILD_DIR/configure.in
+++ b/heacore/BUILD_DIR/configure.in
@@ -952,6 +952,7 @@ PYTHON_LIB=""
 PY_CONFIG=""
 NUMPY_INC=""
 AC_PATH_PROGS(PYTHON, python3 python)
+PYTHON=""
 if test "x$PYTHON" = x -o ! -x "$PYTHON"; then
   AC_MSG_WARN(Python executable $PYTHON not found or unusable - not building HEASoftPy or the HEASP Python interface)
   PYTHON=""

