diff --git a/heacore/BUILD_DIR/configure.in b/heacore/BUILD_DIR/configure.in
index 958c18c..2364087 100644
--- a/heacore/BUILD_DIR/configure.in
+++ b/heacore/BUILD_DIR/configure.in
@@ -952,6 +952,7 @@ PYTHON_LIB=""
 PY_CONFIG=""
 NUMPY_INC=""
 AC_PATH_PROGS(PYTHON, python3 python)
+PYTHON=""
 if test "x$PYTHON" = x -o ! -x "$PYTHON"; then
   AC_MSG_WARN(Python executable $PYTHON not found or unusable - not building HEASoftPy or the HEASP Python interface)
   PYTHON=""
diff --git a/BUILD_DIR/configure.in b/BUILD_DIR/configure.in
index 7ba0458..71e66e1 100644
--- a/BUILD_DIR/configure.in
+++ b/BUILD_DIR/configure.in
@@ -844,7 +844,7 @@ h_Xspec_components="src/xslib src/XSUtil src/XSFunctions src/XSModel src/XSStat
 if test "$h_xs_models_only" = yes; then
   h_heacore_components="gsl fgsl fftw cfitsio ape heaio heautils wcslib CCfits heasp"
   h_tcltk_components=""
-  h_Xspec_components="src/xslib src/XSUtil src/XSFunctions mpython"
+  h_Xspec_components="src/xslib src/XSUtil src/XSFunctions src/XSModel mpython"
   #force_readline_build=no
   HD_SELECTED="heacore Xspec"
   HD_SELECTED_DIRS="$h_top/heacore/BUILD_DIR $h_top/Xspec/BUILD_DIR"
